builtin/logical/transit/backend.go

```diff
@@ -68,6 +68,12 @@ func Backend(ctx context.Context, conf *logical.BackendConfig) (*backend, error)
 		if err != nil {
 			return nil, fmt.Errorf("Error retrieving cache size from storage: %w", err)
 		}
+
+		minCacheSize := 10
```

Comment:
Since this is a constant, break it out as a Go "const" at the beginning of the file.  You can find examples of this in other code.